azure-proto-authorization/RoleAssignmentOperations.cs

```diff
@@ -48,6 +48,8 @@ internal RoleAssignmentOperations(AzureResourceManagerClientOptions options, Res
 
         private RoleAssignmentsOperations Operations => GetClient<AuthorizationManagementClient>((baseUri, creds) => new AuthorizationManagementClient(Id.Subscription, baseUri, creds)).RoleAssignments;
 
+        protected override ResourceType ValidResourceType => ResourceType;
```

Comment:
nit, I think the protected needs to be above private.

Reply:
other way around